doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -122,15 +122,15 @@ when anomaly logging is enabled.
 Fields
 ------
 
-* "type": Either "packet", "stream" or "applayer". In rare cases, type will be "unknown".
+* "type": Either "decode", "stream" or "application-layer". In rare cases, type will be "unknown".
```

Comment:
why the rename from ```applayer``` to ```application-layer```? I like the short version much better.

Reply:
Will change to use `applayer`